libs/challenge-registry/ui/src/lib/navbar/navbar.component.html

```diff
@@ -24,7 +24,7 @@
     >Log In</a
   >
   <a mat-button class="sage-button" *ngIf="!isLoggedIn" routerLink="/signup" aria-label="Sign Up"
-    >Sign Up</a
+    >Join OC</a
```

Comment:
"OC" looks good to me, as long as we will mention what stands for, somewhere in the home page maybe.

Reply:
"OC" sounds weird to me, maybe because we haven't used this yet. Also it should be evident that the Register / Join button is about joining OpenChallenges.

Another option is also to remove this button from the navbar and only include "Log in", because a login page always has a link like "Not registered yet? Create an account".

Reply:
Oh whoops, "OC" was not meant to be pushed.  Sorry about that!

Reply:
> Another option is also to remove this button from the navbar and only include "Log in", because a login page always has a link like "Not registered yet? Create an account".

This sounds reasonable to me.  I'd like to talk with Adam too to see what he thinks from a UX standpoint.

Reply:
Perfect, thanks!

Reply:
Just spoke with Adam -- he agrees that users will get the general flow if we take out "Join"/"Register" from the navbar!

Will implement now.